examples/example_group_convolution.cc

```diff
@@ -340,7 +340,7 @@ TEST_F(GroupConvolution, ATenGroupConvolutionReference) {
           auto weight_g = subtensor(W2, 0, G, g);
           auto bias_g = subtensor(B1, 0, G, g);
           outputs[g] =
-              at::conv2d(input_g, weight_g, at::IntList({KH, KW}), bias_g);
+              at::thnn_conv2d(input_g, weight_g, at::IntList({KH, KW}), bias_g);
```

Comment:
There’s no reason to use the thnn impl. Just remove the IntList arg

Reply:
cool, thanks @apaszke , will do so.

Reply:
@apaszke , it looks like `conv2d` doesn't take kernel size as argument https://github.com/zdevito/ATen/blob/d03c3e37b62827865ba210b31528d9e00024cf7f/src/ATen/native/Convolution.cpp#L242-L247 
but `thnn_conv2d` does https://github.com/zdevito/ATen/blob/85f56d1adce2ed0b1f7859db399bfc2e8c26a461/src/ATen/nn.yaml#L271-L273 . do you know how `conv2d` handles the kernel size argument?

Reply:
It's redundant, so it is inferred from the weights.